releasenotes/notes/enable-telemetry-by-default-ae39651a478f8e61.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Enable :ref:`telemetry <Instrumentation Telemetry>` collection when using ddtrace-run.
```

```suggestion
    Enable :ref:`telemetry <Instrumentation Telemetry>` collection when using ``ddtrace-run``.
```

Does spelling complain about this? not sure this suggestion matters